staging/src/k8s.io/apiserver/pkg/util/flowcontrol/reqmgmt.go

```diff
@@ -154,19 +151,17 @@ func NewRequestManagementSystem(
 // optional and, if supplied, is kept informed of whenever a goroutine
 // is started or stopped or begins or finishes waiting --- except that
 // the configuration controller is not fully plumbed yet.
-func NewRequestManagementSystemWithPreservation(
+func NewRequestManagerWithPreservation(
 	informerFactory kubeinformers.SharedInformerFactory,
 	flowcontrolClient rmclientv1alpha1.FlowcontrolV1alpha1Interface,
-	queueSetFactory fq.QueueSetFactory,
```

Comment:
Removing these two parameters makes this function not good for unit testing, the unit tests will have to construct `requestManagementSystem` structs directly.  Is this an improvement?